roachtest: tpcc/multiregion/survive=region/chaos=true failed

roachtest.tpcc/multiregion/survive=region/chaos=true [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3653448&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3653448&tab=artifacts#/tpcc/multiregion/survive=region/chaos=true) on master @ [40bd241158c8dd4a68d277fa732b91f56e411b38](https://github.com/cockroachdb/cockroach/commits/40bd241158c8dd4a68d277fa732b91f56e411b38):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/tpcc/multiregion/survive=region/chaos=true/run_1
	tpcc.go:256,tpcc.go:565,test_runner.go:777: error at from 2021-10-29T13:30:57Z, to 2021-10-29T13:35:37Z on metric workload_tpcc_newOrder_error_total{instance="35.237.23.22:2110"}: expected <=100 errors, found from 617689.000000, to 617912.000000
		(1) secondary error attachment
		  | error at from 2021-10-29T14:01:05Z, to 2021-10-29T14:05:45Z on metric workload_tpcc_newOrder_error_total{instance="35.237.23.22:2110"}: expected <=100 errors, found from 1241028.000000, to 1241189.000000
		  | (1) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:207
		  |   | [...repeated from below...]
		  | Wraps: (2) error at from 2021-10-29T14:01:05Z, to 2021-10-29T14:05:45Z on metric workload_tpcc_newOrder_error_total{instance="35.237.23.22:2110"}
		  | Wraps: (3) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:78
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:206
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:56
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:282
		  |   | runtime.goexit
		  |   | 	/usr/local/go/src/runtime/asm_amd64.s:1371
		  | Wraps: (4) expected <=100 errors, found from 1241028.000000, to 1241189.000000
		  | Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.leafError
		Wraps: (2) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:207
		  | [...repeated from below...]
		Wraps: (3) error at from 2021-10-29T13:30:57Z, to 2021-10-29T13:35:37Z on metric workload_tpcc_newOrder_error_total{instance="35.237.23.22:2110"}
		Wraps: (4) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:78
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:206
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:56
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:282
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1371
		Wraps: (5) expected <=100 errors, found from 617689.000000, to 617912.000000
		Error types: (1) *secondary.withSecondaryError (2) *withstack.withStack (3) *errutil.withPrefix (4) *withstack.withStack (5) *errutil.leafError
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

 | 

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
/cc @cockroachdb/multiregion
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*tpcc/multiregion/survive=region/chaos=true.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.tpcc/multiregion/survive=region/chaos=true [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3658629&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3658629&tab=artifacts#/tpcc/multiregion/survive=region/chaos=true) on master @ [db229cad354c2776ea35c9dc4f78e6ce66b9437c](https://github.com/cockroachdb/cockroach/commits/db229cad354c2776ea35c9dc4f78e6ce66b9437c):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/tpcc/multiregion/survive=region/chaos=true/run_1
	tpcc.go:256,tpcc.go:565,test_runner.go:777: error at from 2021-10-30T12:37:29Z, to 2021-10-30T12:42:09Z on metric workload_tpcc_newOrder_error_total{instance="35.229.44.35:2110"}: expected <=100 errors, found from 655216.000000, to 655491.000000
		(1) secondary error attachment
		  | error at from 2021-10-30T13:07:37Z, to 2021-10-30T13:12:17Z on metric workload_tpcc_newOrder_error_total{instance="35.229.44.35:2110"}: expected <=100 errors, found from 1347774.000000, to 1348004.000000
		  | (1) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:208
		  |   | [...repeated from below...]
		  | Wraps: (2) error at from 2021-10-30T13:07:37Z, to 2021-10-30T13:12:17Z on metric workload_tpcc_newOrder_error_total{instance="35.229.44.35:2110"}
		  | Wraps: (3) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:79
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:207
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:57
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:283
		  |   | runtime.goexit
		  |   | 	/usr/local/go/src/runtime/asm_amd64.s:1371
		  | Wraps: (4) expected <=100 errors, found from 1347774.000000, to 1348004.000000
		  | Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.leafError
		Wraps: (2) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:208
		  | [...repeated from below...]
		Wraps: (3) error at from 2021-10-30T12:37:29Z, to 2021-10-30T12:42:09Z on metric workload_tpcc_newOrder_error_total{instance="35.229.44.35:2110"}
		Wraps: (4) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:79
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:207
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:57
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:283
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1371
		Wraps: (5) expected <=100 errors, found from 655216.000000, to 655491.000000
		Error types: (1) *secondary.withSecondaryError (2) *withstack.withStack (3) *errutil.withPrefix (4) *withstack.withStack (5) *errutil.leafError
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

 | 

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
/cc @cockroachdb/multiregion
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*tpcc/multiregion/survive=region/chaos=true.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.tpcc/multiregion/survive=region/chaos=true [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3667494&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3667494&tab=artifacts#/tpcc/multiregion/survive=region/chaos=true) on master @ [f69253e3d1cfe798821ea1b1c85aeb79e9eece32](https://github.com/cockroachdb/cockroach/commits/f69253e3d1cfe798821ea1b1c85aeb79e9eece32):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/tpcc/multiregion/survive=region/chaos=true/run_1
	tpcc.go:256,tpcc.go:565,test_runner.go:777: error at from 2021-11-02T13:02:24Z, to 2021-11-02T13:07:04Z on metric workload_tpcc_newOrder_error_total{instance="35.237.106.28:2110"}: expected <=100 errors, found from 580316.000000, to 580535.000000
		(1) secondary error attachment
		  | error at from 2021-11-02T13:32:33Z, to 2021-11-02T13:37:13Z on metric workload_tpcc_newOrder_error_total{instance="35.237.106.28:2110"}: expected <=100 errors, found from 1202780.000000, to 1202972.000000
		  | (1) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:208
		  |   | [...repeated from below...]
		  | Wraps: (2) error at from 2021-11-02T13:32:33Z, to 2021-11-02T13:37:13Z on metric workload_tpcc_newOrder_error_total{instance="35.237.106.28:2110"}
		  | Wraps: (3) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:79
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:207
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:57
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:283
		  |   | runtime.goexit
		  |   | 	/usr/local/go/src/runtime/asm_amd64.s:1371
		  | Wraps: (4) expected <=100 errors, found from 1202780.000000, to 1202972.000000
		  | Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.leafError
		Wraps: (2) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:208
		  | [...repeated from below...]
		Wraps: (3) error at from 2021-11-02T13:02:24Z, to 2021-11-02T13:07:04Z on metric workload_tpcc_newOrder_error_total{instance="35.237.106.28:2110"}
		Wraps: (4) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:79
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:207
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:57
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:283
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1371
		Wraps: (5) expected <=100 errors, found from 580316.000000, to 580535.000000
		Error types: (1) *secondary.withSecondaryError (2) *withstack.withStack (3) *errutil.withPrefix (4) *withstack.withStack (5) *errutil.leafError
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

 | 

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
/cc @cockroachdb/multiregion
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*tpcc/multiregion/survive=region/chaos=true.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.tpcc/multiregion/survive=region/chaos=true [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3674519&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3674519&tab=artifacts#/tpcc/multiregion/survive=region/chaos=true) on master @ [05d36621ae805e644d98ef6b35d453ae16255ccc](https://github.com/cockroachdb/cockroach/commits/05d36621ae805e644d98ef6b35d453ae16255ccc):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/tpcc/multiregion/survive=region/chaos=true/run_1
	tpcc.go:256,tpcc.go:565,test_runner.go:777: error at from 2021-11-03T15:37:20Z, to 2021-11-03T15:42:00Z on metric workload_tpcc_newOrder_error_total{instance="35.227.30.152:2110"}: expected <=100 errors, found from 611647.000000, to 611861.000000
		(1) secondary error attachment
		  | error at from 2021-11-03T16:07:29Z, to 2021-11-03T16:12:09Z on metric workload_tpcc_newOrder_error_total{instance="35.227.30.152:2110"}: expected <=100 errors, found from 1236924.000000, to 1237136.000000
		  | (1) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:208
		  |   | [...repeated from below...]
		  | Wraps: (2) error at from 2021-11-03T16:07:29Z, to 2021-11-03T16:12:09Z on metric workload_tpcc_newOrder_error_total{instance="35.227.30.152:2110"}
		  | Wraps: (3) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:79
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:207
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:57
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:283
		  |   | runtime.goexit
		  |   | 	/usr/local/go/src/runtime/asm_amd64.s:1371
		  | Wraps: (4) expected <=100 errors, found from 1236924.000000, to 1237136.000000
		  | Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.leafError
		Wraps: (2) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:208
		  | [...repeated from below...]
		Wraps: (3) error at from 2021-11-03T15:37:20Z, to 2021-11-03T15:42:00Z on metric workload_tpcc_newOrder_error_total{instance="35.227.30.152:2110"}
		Wraps: (4) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:79
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:207
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:57
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:283
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1371
		Wraps: (5) expected <=100 errors, found from 611647.000000, to 611861.000000
		Error types: (1) *secondary.withSecondaryError (2) *withstack.withStack (3) *errutil.withPrefix (4) *withstack.withStack (5) *errutil.leafError
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

 | 

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
/cc @cockroachdb/multiregion
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*tpcc/multiregion/survive=region/chaos=true.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.tpcc/multiregion/survive=region/chaos=true [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3686697&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3686697&tab=artifacts#/tpcc/multiregion/survive=region/chaos=true) on master @ [ed35bbe8f6e5947db6e3f2423d1aa9dde1d15804](https://github.com/cockroachdb/cockroach/commits/ed35bbe8f6e5947db6e3f2423d1aa9dde1d15804):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/tpcc/multiregion/survive=region/chaos=true/run_1
	tpcc.go:256,tpcc.go:565,test_runner.go:777: error at from 2021-11-05T14:04:15Z, to 2021-11-05T14:08:55Z on metric workload_tpcc_newOrder_error_total{instance="34.139.149.229:2120"}: expected <=100 errors, found from 1867502.000000, to 1867607.000000
		(1) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:208
		  | [...repeated from below...]
		Wraps: (2) error at from 2021-11-05T14:04:15Z, to 2021-11-05T14:08:55Z on metric workload_tpcc_newOrder_error_total{instance="34.139.149.229:2120"}
		Wraps: (3) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:79
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:207
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:57
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:283
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1371
		Wraps: (4) expected <=100 errors, found from 1867502.000000, to 1867607.000000
		Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.leafError
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

 | 

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
/cc @cockroachdb/multiregion
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*tpcc/multiregion/survive=region/chaos=true.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.tpcc/multiregion/survive=region/chaos=true [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3692874&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3692874&tab=artifacts#/tpcc/multiregion/survive=region/chaos=true) on master @ [a0849b11188efd1c58dc339d15f75b03e0762459](https://github.com/cockroachdb/cockroach/commits/a0849b11188efd1c58dc339d15f75b03e0762459):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/tpcc/multiregion/survive=region/chaos=true/run_1
	tpcc.go:256,tpcc.go:565,test_runner.go:777: error at from 2021-11-08T14:26:45Z, to 2021-11-08T14:31:25Z on metric workload_tpcc_newOrder_error_total{instance="34.73.197.61:2110"}: expected <=100 errors, found from 551259.000000, to 551502.000000
		(1) secondary error attachment
		  | error at from 2021-11-08T14:56:54Z, to 2021-11-08T15:01:34Z on metric workload_tpcc_newOrder_error_total{instance="34.73.197.61:2110"}: expected <=100 errors, found from 1123423.000000, to 1123626.000000
		  | (1) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:208
		  |   | [...repeated from below...]
		  | Wraps: (2) error at from 2021-11-08T14:56:54Z, to 2021-11-08T15:01:34Z on metric workload_tpcc_newOrder_error_total{instance="34.73.197.61:2110"}
		  | Wraps: (3) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:79
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:207
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:57
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:283
		  |   | runtime.goexit
		  |   | 	/usr/local/go/src/runtime/asm_amd64.s:1371
		  | Wraps: (4) expected <=100 errors, found from 1123423.000000, to 1123626.000000
		  | Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.leafError
		Wraps: (2) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:208
		  | [...repeated from below...]
		Wraps: (3) error at from 2021-11-08T14:26:45Z, to 2021-11-08T14:31:25Z on metric workload_tpcc_newOrder_error_total{instance="34.73.197.61:2110"}
		Wraps: (4) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:79
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:207
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:57
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:283
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1371
		Wraps: (5) expected <=100 errors, found from 551259.000000, to 551502.000000
		Error types: (1) *secondary.withSecondaryError (2) *withstack.withStack (3) *errutil.withPrefix (4) *withstack.withStack (5) *errutil.leafError
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

 | 

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
/cc @cockroachdb/multiregion
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*tpcc/multiregion/survive=region/chaos=true.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.tpcc/multiregion/survive=region/chaos=true [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3705715&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3705715&tab=artifacts#/tpcc/multiregion/survive=region/chaos=true) on master @ [2de17e7fbe66e14039fc7969a76139625761438f](https://github.com/cockroachdb/cockroach/commits/2de17e7fbe66e14039fc7969a76139625761438f):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/tpcc/multiregion/survive=region/chaos=true/run_1
	tpcc.go:256,tpcc.go:565,test_runner.go:777: error at from 2021-11-10T14:14:46Z, to 2021-11-10T14:19:26Z on metric workload_tpcc_newOrder_error_total{instance="35.196.30.146:2110"}: expected <=100 errors, found from 643648.000000, to 643880.000000
		(1) secondary error attachment
		  | error at from 2021-11-10T14:44:55Z, to 2021-11-10T14:49:35Z on metric workload_tpcc_newOrder_error_total{instance="35.196.30.146:2110"}: expected <=100 errors, found from 1288117.000000, to 1288286.000000
		  | (1) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:208
		  |   | [...repeated from below...]
		  | Wraps: (2) error at from 2021-11-10T14:44:55Z, to 2021-11-10T14:49:35Z on metric workload_tpcc_newOrder_error_total{instance="35.196.30.146:2110"}
		  | Wraps: (3) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:79
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:207
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:57
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:283
		  |   | runtime.goexit
		  |   | 	/usr/local/go/src/runtime/asm_amd64.s:1371
		  | Wraps: (4) expected <=100 errors, found from 1288117.000000, to 1288286.000000
		  | Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.leafError
		Wraps: (2) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:208
		  | [...repeated from below...]
		Wraps: (3) error at from 2021-11-10T14:14:46Z, to 2021-11-10T14:19:26Z on metric workload_tpcc_newOrder_error_total{instance="35.196.30.146:2110"}
		Wraps: (4) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:79
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:207
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:57
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:283
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1371
		Wraps: (5) expected <=100 errors, found from 643648.000000, to 643880.000000
		Error types: (1) *secondary.withSecondaryError (2) *withstack.withStack (3) *errutil.withPrefix (4) *withstack.withStack (5) *errutil.leafError
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

 | 

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
/cc @cockroachdb/multiregion
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*tpcc/multiregion/survive=region/chaos=true.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.tpcc/multiregion/survive=region/chaos=true [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3721766&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3721766&tab=artifacts#/tpcc/multiregion/survive=region/chaos=true) on master @ [d16a755cfa43e10a85e4c9aa9400b5a147b65e69](https://github.com/cockroachdb/cockroach/commits/d16a755cfa43e10a85e4c9aa9400b5a147b65e69):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/tpcc/multiregion/survive=region/chaos=true/run_1
	tpcc.go:256,tpcc.go:565,test_runner.go:777: error at from 2021-11-12T23:20:31Z, to 2021-11-12T23:25:11Z on metric workload_tpcc_newOrder_error_total{instance="35.237.178.1:2110"}: expected <=100 errors, found from 629341.000000, to 629481.000000
		(1) secondary error attachment
		  | error at from 2021-11-12T23:30:34Z, to 2021-11-12T23:35:14Z on metric workload_tpcc_newOrder_error_total{instance="35.237.178.1:2111"}: expected <=100 errors, found from 412200.000000, to 412314.000000
		  | (1) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:208
		  |   | [...repeated from below...]
		  | Wraps: (2) error at from 2021-11-12T23:30:34Z, to 2021-11-12T23:35:14Z on metric workload_tpcc_newOrder_error_total{instance="35.237.178.1:2111"}
		  | Wraps: (3) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:79
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:207
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:57
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:283
		  |   | runtime.goexit
		  |   | 	/usr/local/go/src/runtime/asm_amd64.s:1371
		  | Wraps: (4) expected <=100 errors, found from 412200.000000, to 412314.000000
		  | Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.leafError
		Wraps: (2) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:208
		  | [...repeated from below...]
		Wraps: (3) error at from 2021-11-12T23:20:31Z, to 2021-11-12T23:25:11Z on metric workload_tpcc_newOrder_error_total{instance="35.237.178.1:2110"}
		Wraps: (4) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:79
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:207
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:57
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:283
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1371
		Wraps: (5) expected <=100 errors, found from 629341.000000, to 629481.000000
		Error types: (1) *secondary.withSecondaryError (2) *withstack.withStack (3) *errutil.withPrefix (4) *withstack.withStack (5) *errutil.leafError
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*tpcc/multiregion/survive=region/chaos=true.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.tpcc/multiregion/survive=region/chaos=true [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3738783&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3738783&tab=artifacts#/tpcc/multiregion/survive=region/chaos=true) on master @ [e921036c0640d833548363cd8f3fea78ae534bd1](https://github.com/cockroachdb/cockroach/commits/e921036c0640d833548363cd8f3fea78ae534bd1):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/tpcc/multiregion/survive=region/chaos=true/run_1
	tpcc.go:256,tpcc.go:565,test_runner.go:777: error at from 2021-11-16T13:40:56Z, to 2021-11-16T13:45:36Z on metric workload_tpcc_newOrder_error_total{instance="35.237.147.124:2110"}: expected <=100 errors, found from 650508.000000, to 650734.000000
		(1) secondary error attachment
		  | error at from 2021-11-16T14:11:05Z, to 2021-11-16T14:15:45Z on metric workload_tpcc_newOrder_error_total{instance="35.237.147.124:2110"}: expected <=100 errors, found from 1301613.000000, to 1301901.000000
		  | (1) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:208
		  |   | [...repeated from below...]
		  | Wraps: (2) error at from 2021-11-16T14:11:05Z, to 2021-11-16T14:15:45Z on metric workload_tpcc_newOrder_error_total{instance="35.237.147.124:2110"}
		  | Wraps: (3) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:79
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:207
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:57
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:283
		  |   | runtime.goexit
		  |   | 	/usr/local/go/src/runtime/asm_amd64.s:1371
		  | Wraps: (4) expected <=100 errors, found from 1301613.000000, to 1301901.000000
		  | Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.leafError
		Wraps: (2) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:208
		  | [...repeated from below...]
		Wraps: (3) error at from 2021-11-16T13:40:56Z, to 2021-11-16T13:45:36Z on metric workload_tpcc_newOrder_error_total{instance="35.237.147.124:2110"}
		Wraps: (4) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:79
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:207
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:57
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:283
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1371
		Wraps: (5) expected <=100 errors, found from 650508.000000, to 650734.000000
		Error types: (1) *secondary.withSecondaryError (2) *withstack.withStack (3) *errutil.withPrefix (4) *withstack.withStack (5) *errutil.leafError
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*tpcc/multiregion/survive=region/chaos=true.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.tpcc/multiregion/survive=region/chaos=true [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3770302&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3770302&tab=artifacts#/tpcc/multiregion/survive=region/chaos=true) on master @ [6abeda76d5ed629cf52aeac2e943aa55cf7212fc](https://github.com/cockroachdb/cockroach/commits/6abeda76d5ed629cf52aeac2e943aa55cf7212fc):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/tpcc/multiregion/survive=region/chaos=true/run_1
	tpcc.go:256,tpcc.go:565,test_runner.go:777: error at from 2021-11-21T14:21:48Z, to 2021-11-21T14:26:28Z on metric workload_tpcc_newOrder_error_total{instance="34.73.238.179:2110"}: expected <=100 errors, found from 601525.000000, to 601754.000000
		(1) secondary error attachment
		  | error at from 2021-11-21T14:51:57Z, to 2021-11-21T14:56:37Z on metric workload_tpcc_newOrder_error_total{instance="34.73.238.179:2110"}: expected <=100 errors, found from 1201440.000000, to 1201637.000000
		  | (1) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:208
		  |   | [...repeated from below...]
		  | Wraps: (2) error at from 2021-11-21T14:51:57Z, to 2021-11-21T14:56:37Z on metric workload_tpcc_newOrder_error_total{instance="34.73.238.179:2110"}
		  | Wraps: (3) attached stack trace
		  |   -- stack trace:
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:79
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:207
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:57
		  |   | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  |   | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:283
		  |   | runtime.goexit
		  |   | 	/usr/local/go/src/runtime/asm_amd64.s:1581
		  | Wraps: (4) expected <=100 errors, found from 1201440.000000, to 1201637.000000
		  | Error types: (1) *withstack.withStack (2) *errutil.withPrefix (3) *withstack.withStack (4) *errutil.leafError
		Wraps: (2) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:208
		  | [...repeated from below...]
		Wraps: (3) error at from 2021-11-21T14:21:48Z, to 2021-11-21T14:26:28Z on metric workload_tpcc_newOrder_error_total{instance="34.73.238.179:2110"}
		Wraps: (4) attached stack trace
		  -- stack trace:
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime.func2
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:79
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkMetrics
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:207
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).checkUptime
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:57
		  | github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests.(*tpccChaosEventProcessor).listen.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/drt.go:283
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1581
		Wraps: (5) expected <=100 errors, found from 601525.000000, to 601754.000000
		Error types: (1) *secondary.withSecondaryError (2) *withstack.withStack (3) *errutil.withPrefix (4) *withstack.withStack (5) *errutil.leafError
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*tpcc/multiregion/survive=region/chaos=true.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/tpcc.go
+++ b/pkg/cmd/roachtest/tests/tpcc.go
@@ -617,7 +617,7 @@ func registerTPCC(r registry.Registry) {
 								// * ERROR: inbox communication error: rpc error: code = Canceled
 								//   desc = context canceled (SQLSTATE 58C01)
 								// Setting this allows some errors to occur.
-								maxErrorsDuringUptime: warehousesPerRegion * 5,
+								maxErrorsDuringUptime: warehousesPerRegion * tpcc.NumWorkersPerWarehouse,
 								// "delivery" does not trigger often.
 								allowZeroSuccessDuringUptime: true,
 							}, nil

--- a/pkg/workload/tpcc/tpcc.go
+++ b/pkg/workload/tpcc/tpcc.go
@@ -192,7 +192,7 @@ var tpccMeta = workload.Meta{
 		g.flags.StringVar(&g.dbOverride, `db`, ``,
 			`Override for the SQL database to use. If empty, defaults to the generator name`)
 		g.flags.IntVar(&g.workers, `workers`, 0, fmt.Sprintf(
-			`Number of concurrent workers. Defaults to --warehouses * %d`, numWorkersPerWarehouse,
+			`Number of concurrent workers. Defaults to --warehouses * %d`, NumWorkersPerWarehouse,
 		))
 		g.flags.IntVar(&g.numConns, `conns`, 0, fmt.Sprintf(
 			`Number of connections. Defaults to --warehouses * %d (except in nowait mode, where it defaults to --workers`,
@@ -319,7 +319,7 @@ func (w *tpcc) Hooks() workload.Hooks {
 			w.initNonUniformRandomConstants()
 
 			if w.workers == 0 {
-				w.workers = w.activeWarehouses * numWorkersPerWarehouse
+				w.workers = w.activeWarehouses * NumWorkersPerWarehouse
 			}
 
 			if w.numConns == 0 {
@@ -334,9 +334,9 @@ func (w *tpcc) Hooks() workload.Hooks {
 				}
 			}
 
-			if w.waitFraction > 0 && w.workers != w.activeWarehouses*numWorkersPerWarehouse {
+			if w.waitFraction > 0 && w.workers != w.activeWarehouses*NumWorkersPerWarehouse {
 				return errors.Errorf(`--wait > 0 and --warehouses=%d requires --workers=%d`,
-					w.activeWarehouses, w.warehouses*numWorkersPerWarehouse)
+					w.activeWarehouses, w.warehouses*NumWorkersPerWarehouse)
 			}
 
 			if w.serializable {

--- a/pkg/workload/tpcc/worker.go
+++ b/pkg/workload/tpcc/worker.go
@@ -28,7 +28,8 @@ import (
 )
 
 const (
-	numWorkersPerWarehouse = 10
+	// NumWorkersPerWarehouse is the default number of workers per warehouse.
+	NumWorkersPerWarehouse = 10
 	numConnsPerWarehouse   = 2
 )
 

